src/components/Comments/WriteComment.js

```diff
@@ -241,7 +242,7 @@ export const styles = StyleSheet.create({
   connectorBottom: (inReplyCommentView, showProfileConnector) => ({
     height: showProfileConnector ? 20 : 0,
     width: 1,
-    backgroundColor: COLORS.gray9,
+    backgroundColor: colors.gray1,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the connectors has been updated. Ensure this color change aligns with your design guidelines.
```diff
- backgroundColor: COLORS.gray9,
+ backgroundColor: colors.gray1,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->